students/minjeong/users/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+# Create your models here.
+class Signup(models.Model):
+    name=models.CharField(max_length=45)
+    email=models.EmailField(max_length=45, unique=True)
```

Comment:
- 질문에 대한 답으로 EmailField로 작성하고 Validator를 추가해줄 수도 있지만, Django에 의존하지 않고 직접 정규표현식을 이용하기 위해 CharField 사용해주세요.
```suggestion
    email=models.CharField(max_length=45, unique=True)
```

Reply:
네